docs/network-flow-visibility.md

```diff
@@ -367,7 +367,11 @@ git clone --depth 1 --branch main https://github.com/vmware-tanzu/antrea.git &&
 or
 svn export https://github.com/vmware-tanzu/antrea/trunk/build/yamls/elk-flow-collector/
 ```
-
+To configure the export interval as `flowExportInterval` in [Configuration](#configuration), 
+modify following `interval` value (in seconds) in `elk-flow-collector/logstash/logstash.conf`
+```
+script_params => { "interval" => 60 }
```

Comment:
With dynamic interval tracking for flow records, is this still valid?

Reply:
It is valid for the first record.